source/vibe/http/router.d

```diff
@@ -136,6 +136,8 @@ final class URLRouter : HTTPServerRequestHandler {
 		else m_routes ~= Route(method, path, cb);
 		return this;
 	}
+	/// Adds a new route for request that match the path and method
+	HTTPRouter match(HTTPMethod method, string path, HTTPServerRequestDelegate cb);
```

Comment:
Wait, I think those two lines shouldn't be in this diff at all, right? It's the same signature as above, but returns the deprecated `HTTPRouter` interfaces and has no implementation.